TSOMessageHub/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Quartz;
+using TSOMessageHub;
+using MassTransit;
+using RabbitMQ.Client;
+using TSOMessageHub.Consumers;
+using TSOMessageHub.XML;
+
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices((host, services) =>
+    {
+        services.AddMassTransit(x =>
+        {
+            x.UsingRabbitMq((context, cfg) =>
+            {
+                cfg.Host(host.Configuration["RabbitMQHost"], 5672, "ortfmwaf", h => {
+                    h.Username(host.Configuration["RabbitMQUsername"]);
+                    h.Password(host.Configuration["RabbitMQPassword"]);
+                });
+                // Uncomment in case you want to debug and consume your own messages
+                /*x.AddConsumer<SignalConsumer>(c =>
+                {
+                    c.UseConcurrencyLimit(1);
+                    c.UseMessageRetry(f => f.Intervals(TimeSpan.FromMilliseconds(250), TimeSpan.FromMilliseconds(500), TimeSpan.FromMilliseconds(1000)));
+                });
+                cfg.ReceiveEndpoint("tso-signal-list", x =>
+                {
+                    x.ConfigureConsumeTopology = false;
+                    x.Bind("TSOMessageHub.XML:AfrrSignal", s =>
+                    {
+                        s.RoutingKey = "AfrrSignal";
+                        s.ExchangeType = ExchangeType.Topic;
+                    });
+                    x.ConfigureConsumer<SignalConsumer>(context);
+                });*/
```

Comment:
Move the x.AddConsumer back inside the servicer.AddMassTransit instead of in x.UsingRabbitMq as it crashes otherwise
```suggestion
            // [1] Uncomment, as well as [2], in case you want to debug and consume your own messages
            /*x.AddConsumer<SignalConsumer>(c =>
            {
                c.UseConcurrencyLimit(1);
                c.UseMessageRetry(f => f.Intervals(TimeSpan.FromMilliseconds(250), TimeSpan.FromMilliseconds(500), TimeSpan.FromMilliseconds(1000)));
            });*/
            x.UsingRabbitMq((context, cfg) =>
            {
                cfg.Host(host.Configuration["RabbitMQHost"], 5672, "ortfmwaf", h => {
                    h.Username(host.Configuration["RabbitMQUsername"]);
                    h.Password(host.Configuration["RabbitMQPassword"]);
                });
                // [2] Uncomment, as well as [1] in case you want to debug and consume your own messages
                /*cfg.ReceiveEndpoint("tso-signal-list", x =>
                {
                    x.ConfigureConsumeTopology = false;
                    x.Bind("TSOMessageHub.XML:AfrrSignal", s =>
                    {
                        s.RoutingKey = "AfrrSignal";
                        s.ExchangeType = ExchangeType.Topic;
                    });
                    x.ConfigureConsumer<SignalConsumer>(context);
                });*/
```